storage/pebble/blocks.go

```diff
@@ -253,6 +253,15 @@ func (b *Blocks) getBlock(keyCode byte, key []byte) (*models.Block, error) {
 	}
 
 	if b.chainID == flowGo.Testnet && slices.Contains(testnetBrokenParentHashBlockHeights, block.Height) {
```

Comment:
Do we have the same issue for mainnet? 

Do we want to define `brokenParentHashBlockHeights`, and for testnet and mainnet, we use different heights, so that the logic to handle the heights can be reused.

Reply:
This issue is specific to `testnet`, because of the introduction of the `PrevRandao` field on `EVM.BlockExecuted` event, and the `Block` type. On `mainnet`, the `PrevRandao` field was present since launch, so this issue does not apply there.